javascript/src/async/point.js

```diff
@@ -2,6 +2,7 @@ import resync from '../resync'
 import { setElevation } from '../actions/point'
 import { setRegion, requestRegions, receiveRegions } from '../actions/region'
 import { urlEncode } from '../io'
+import config from '../../../../javascript/src/seedsource/config'
```

Comment:
This should just be:

```javascript
import config from 'seedsource/config'
```

As it is now, it's always going to import the `seedsource-core` config and never the application-specific override.